### DISCUSSION TITLE
Delay in Receiving GitHub Student Developer Pack Approval

### DISCUSSION THREAD

**Discussion by dasadasanudas** [community/community#147835] [GitHub Education] [answered]
### Hi everyone,

Hi GitHub Support Team,

I am reaching out to inquire about the status of my application for the GitHub Student Developer Pack. I submitted my application on December 19, 2024, and as of today, I have not received any communication regarding its approval.

The resources provided by the GitHub Student Developer Pack are crucial for my academic and personal projects. Considering it has been 5 days since my submission, I am concerned about the delay.

Could someone please provide an update on the status of my application? If there are any additional steps or documentation required from my end, I am more than willing to provide the necessary information to facilitate the approval process.

Thank you for your attention and assistance.

**Comment by dasadasanudas** [comment]
I chose to raise an issue as I saw many people got quickly if they contacted support.

**Comment by Ravindulakmina** [comment]
Dude, go to GitHub support.
[GitHub support](https://support.github.com/request/education)

**Comment by davevad93** [comment]
Hi @dasadasanudas ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @dasadasanudas was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by dasadasanudas** [reply]
Thank you, I already saw it at many places but, "If you haven't received your benefits after approval within 5 days please raise a new discussion so we can address it" 👍, that's why I did, what shall I do, _wait_?

**Reply by davevad93** [reply, marked as answer]
Hey @dasadasanudas yes, you did the right thing rasing a discussion here so now the education team may check it, however, the waiting time isn't always the same for everyone so you shouldn't worry, you will surely get your benefits in a few days. 👍

**Comment by dasadasanudas** [comment]
Thank You to @davevad93,@Ravindulakmina,@davevad93. I finally was verified on Christmas 🤶. Thank You very much to the Github team also for making this **Christmas** very much special to me.

**Reply by Ravindulakmina** [reply]
welcome . best wishes from us
### Merry christmas
